From dff8a3949e855686d871550d786d15f6da221a6a Mon Sep 17 00:00:00 2001
From: Clayton Craft <clayton@craftyguy.net>
Date: Thu, 3 Apr 2025 11:35:46 -0700
Subject: [PATCH 2/2] justfile: install dconf profile

This wasn't being installed by the justfile, so distros using the
justfile for packaging weren't properly installing the dconf profile.

Fixes https://gitlab.alpinelinux.org/alpine/aports/-/issues/17056
---
 Justfile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Justfile b/Justfile
index d348ec8..a3cc10b 100644
--- a/Justfile
+++ b/Justfile
@@ -41,6 +41,9 @@ install:
 	# mimeapps
 	install -Dm0644 data/cosmic-mimeapps.list {{applicationdir}}/cosmic-mimeapps.list
 
+	# dconf profile
+	install -Dm644 data/dconf/profile/cosmic {{cosmic_dconf_profile}}
+
 clean_vendor:
 	rm -rf vendor vendor.tar .cargo/config
 
-- 
2.49.0

